docs/how to/use-transferbench.rst

```diff
@@ -36,71 +39,102 @@ This simplifies to a simple copy operation when dealing with single SRC/DST.
+  The basic specification assumes the same number of SEs used per transfer.
+  A positive number of transfers is specified, followed by the number of SEs and triplets describing each transfer:
 
-   Transfers SEs (srcMem1->Executor1->dstMem1) ... (srcMemL->ExecutorL->dstMemL)
+  Transfers SEs (srcMem1->Executor1->dstMem1) ... (srcMemL->ExecutorL->dstMemL)
 
-2) Advanced
+  The arguments used to specify transfers in the config file are described in the :ref:`arguments table <config_file_arguments_table>`.
 
-   A negative number of Transfers is specified, followed by quintuplets describing each Transfer
-   A non-zero number of bytes specified will override any provided value
+  **Example**:
 
-   -Transfers (srcMem1->Executor1->dstMem1 SEs1 Bytes1) ... (srcMemL->ExecutorL->dstMemL SEsL BytesL)
+  .. code-block:: bash
 
-Argument Details:::
+   1 4 (G0->G0->G1)                   Uses 4 CUs on GPU0 to copy from GPU0 to GPU1
+   1 4 (G2->C1->G0)                   Uses 4 CUs on GPU2 to copy from CPU1 to GPU0
+   2 4 G0->G0->G1 G1->G1->G0          Copies from GPU0 to GPU1, and GPU1 to GPU0, each with 4 SEs
 
-   Transfers:   Number of Transfers to be run in parallel
-   SEs      :   Number of SubExectors to use (CPU threads/ GPU threadblocks)
-   srcMemL   :   Source memory locations (Where the data is to be read from)
-   Executor  :   Executor is specified by a character indicating type, followed by device index (0-indexed)
-                  - C: CPU-executed  (Indexed from 0 to NUMA nodes - 1)
-                  - G: GPU-executed  (Indexed from 0 to GPUs - 1)
-                  - D: DMA-executor  (Indexed from 0 to GPUs - 1)
-   dstMemL   :   Destination memory locations (Where the data is to be written to)
-   bytesL    :   Number of bytes to copy (0 means use command-line specified size)
-                  Must be a multiple of 4 and may be suffixed with ('K','M', or 'G')
+- **Advanced**
 
-                  Memory locations are specified by one or more (device character / device index) pairs
-                  Character indicating memory type followed by device index (0-indexed)
-                  Supported memory locations are:
-                  - C:    Pinned host memory       (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - U:    Unpinned host memory     (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - B:    Fine-grain host memory   (on NUMA node, indexed from 0 to [NUMA nodes-1])
-                  - G:    Global device memory     (on GPU device indexed from 0 to [GPUs - 1])
-                  - F:    Fine-grain device memory (on GPU device indexed from 0 to [GPUs - 1])
-                  - N:    Null memory              (index ignored)
+  A negative number of transfers is specified, followed by quintuplets describing each transfer.
+  A non-zero number of bytes specified, overrides any provided value.
```

Comment:
These two lines are not sentences - something is messed up.

Reply:
Changed it to:
In the advanced specification, a negative number of transfers is specified, followed by quintuplets describing each transfer.
Specifying a non-zero number of bytes overrides any provided value.